### ISSUE TITLE
Allow category descriptions to be updated

### ISSUE THREAD

**Issue by moi90** [ecotaxa/ecotaxa_front#858]
It was proposed to use the PATCH verb to modify the description of a category (by JO, Julie and Bea):

```
PATCH /taxon[/central?]/{taxon_id}]

{source_desc: "..."}
```

This would be helpful to update the descriptions of categories to add [tag-related data](https://github.com/ecotaxa/ecotaxa_front/issues/456#issuecomment-1720903899).

**Comment by grololo06** [comment]
Hello, thanks.
We "just" have to define the security around this function, unfortunately taxa have no "owner", they are created on-demand but then virtually belong to EcoTaxoServer admins.

**Comment by moi90** [comment]
Yes, that is a problem... In fact, the creator is stored. But I would also like to edit categories created by others. And it would also be bad if the EcoTaxoServer admins had to edit hundreds of categories by hand...

**Comment by juliecoust** [comment]
Déplacé sur ClickUp
